kornia/augmentation/random_generator/_2d/random_rain.py

```diff
@@ -41,20 +41,37 @@ def make_samplers(self, device: torch.device, dtype: torch.dtype) -> None:
             bounds=(self.drop_width[0], self.drop_width[1] + 1),
         ).to(device=device, dtype=dtype)
 
```

Comment:
What about dtype ? /cc would make sense to pass device/dtype to the range_bound function until the aright allocation?